<launch>
  <node pkg="cv_camera" type="cv_camera_node" name="camera" />

  <node pkg="opencv3mixing" type="blur_node" name="image_blurer">
    <remap from="input_image" to="camera/image_raw" />
    <remap from="~image_raw" to="image" />
  </node>

  <node pkg="image_view2" type="image_view2" name="view" />
</launch>
